app/models/conversion_host/configurations.rb

```diff
@@ -68,6 +68,8 @@ def enable(params, auth_user = nil)
 
       ssh_key = params.delete(:conversion_host_ssh_private_key)
 
+      osp_ca_bundle = params.delete(:osp_ca_bundle)
```

Comment:
Not sure about this param name. @mturley what say you?

Reply:
I don't have a strong opinion here. @fdupont-redhat, @djberg96 and I briefly discussed using `openstack_tls_ca_certs` to be consistent with the `default_tls_ca_certs` used by classic-ui for RHV. Not sure how much this name matters on the backend, but anything is fine for the UI.

Reply:
Sounds good to me. We have the ems_type as the prefix and a common suffix with RHV.
I've changed it.